### PR TITLE
Install cleanup jeos #8363 ke

### PR DESCRIPTION
**Pull request by keichwa**
https://github.com/SUSE/spacewalk/issues/8363

Cleanup and addressing https://github.com/SUSE/spacewalk/issues/9043

### CODE REVIEW

File: modules/installation/pages/install-vm.adoc [hunk 1]
```diff
@@ -13,7 +13,15 @@ A kernel virtual machine (KVM) combined with Virtual Machine Manager (_virt-mana
 // - 2019-06-19.
 
 Enter the following settings when creating a new virtual machine using *virt-manager*.
+
+[NOTE]
+====
+In the following table minimal requirements are specified, which are suitable for a quick test installation (server with one client).
```
Comment (Loquacity) [inline modules/installation/pages/install-vm.adoc]:
```suggestion
This table specifies the minimum requirements.
These are suitable for a quick test installation, such as a server with one client.
```

File: modules/installation/pages/install-vm.adoc [hunk 2]
```diff
@@ -13,7 +13,15 @@ A kernel virtual machine (KVM) combined with Virtual Machine Manager (_virt-mana
...
+If you want to use it as a production environment, consider the requirements listed in xref:hardware-requirements.adoc[].
```
Comment (Loquacity) [inline modules/installation/pages/install-vm.adoc]:
```suggestion
If you want to use a production environment, review the requirements listed in xref:hardware-requirements.adoc[].
```

File: modules/installation/pages/install-vm.adoc [hunk 3]
```diff
@@ -49,7 +57,7 @@ Create three additional virtual disks required for the {productname} storage par
 . Create a new virtual machine using the downloaded JeOS KVM image and select [guimenu]``Import existing disk image``.
 . Configure RAM and number of CPUs (at least 4 GB RAM and 2 CPUs).
 . Name your KVM machine and select the [guimenu]``Customize configuration before install`` check box.
-. Select the btn:[Add Hardware] button and create three new virtual disks with the following specifications.
+. Click btn:[Add Hardware] and create three new virtual disks with the following specifications.
```
Comment (Loquacity) [inline modules/installation/pages/install-vm.adoc]:
```suggestion
. Click btn:[Add Hardware] to create three new virtual disks with these specifications.
```

File: modules/installation/pages/install-vm.adoc [hunk 4]
```diff
@@ -69,9 +77,9 @@ Also consider to create a separate partition for [path]``/srv`` where Kiwi image
 | VirtIO Disk 4        | swap      | 4{nbsp}GB
 |===
 
-. Click btn:[Begin Installation] to boot the new VM from the JeOS image.
+. To boot the new VM from the JeOS image click btn:[Begin Installation].
```
Comment (Loquacity) [inline modules/installation/pages/install-vm.adoc]:
```suggestion
. Click btn:[Begin Installation] to boot the new VM from the JeOS image.
```
The original was better :stuck_out_tongue:

File: modules/installation/pages/install-vm.adoc [hunk 5]
```diff
@@ -69,9 +77,9 @@ Also consider to create a separate partition for [path]``/srv`` where Kiwi image
...
 
-Proceed through the basic JeOS installation prompts until you reach the command line.
+Proceed through the basic JeOS installation prompts until the command prompt wait for input.
```
Comment (Loquacity) [inline modules/installation/pages/install-vm.adoc]:
```suggestion
Follow the prompts to complete the basic JeOS installation, until the process is complete and the command prompt waits for input.
```
Just trying to clarify this a bit.

File: modules/installation/pages/install-vm.adoc [hunk 6]
```diff
@@ -141,7 +149,7 @@ zypper in -t package yast2-storage-ng
 
 [NOTE]
 ====
-Storage size values are the absolute minimum—only suitable for a small test or demo installation.
+Storage size values are the absolute minimum—suitable only for a small test or demo installation (server and one client).
```
Comment (Loquacity) [inline modules/installation/pages/install-vm.adoc]:
```suggestion
Storage size values are the absolute minimum.
They are suitable only for a small test or demonstration installation, such as a server with one client.
```

File: modules/installation/pages/install-vm.adoc (hunk 1 above)
Comment (keichwa) [reply modules/installation/pages/install-vm.adoc]: Thanks!  So much better

**Comment by keichwa** [comment]
> Fairly minor things.

Nevertheless important.  Every user will read these sections (or should ;) ).

**Pull request merged**